src/SingleMovie/SingleMovie.js

```diff
@@ -9,14 +9,14 @@ const SingleMovie = ({ movieDetails, returnToHome }) => {
       </div>
       <div className="movie-details-holder">
         <h3 className="card-title">{movieDetails.title}</h3>
-        <h3 className="rating">Movie Rating: {Math.round( movieDetails.average_rating * 10 ) / 10}</h3>
+        <h3 className="rating">Rating: {Math.round( movieDetails.average_rating * 10 ) / 10}</h3>
         <h3 className="release-date">Release Date: {movieDetails.release_date}</h3>
-        <h3 className="overview">Movie Overview: {movieDetails.overview}</h3>
+        <h3 className="overview">Overview: {movieDetails.overview}</h3>
```

Comment:
I think we said we will fix the genres so they display with a space in between each element! A small thing to fix later! :)